superset/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -186,7 +186,6 @@ class ChartContainer extends React.Component {
           id={this.props.containerId}
           ref={(ref) => { this.chartContainerRef = ref; }}
           className={this.props.viz_type}
-          style={{ overflowX: 'auto' }}
```

Comment:
Somehow putting it to auto makes the table and pivot table viz show too many scrollbars because of the way the "fixed header" scrolling is implemented